eslint.config.js

```diff
@@ -355,5 +334,30 @@ export default [
 			"max-len": "off",
 			"one-var": "off"
 		}
+	},
+
+	{
+		files: [
+			"dist/jquery.js",
+			"dist/jquery.slim.js",
+			"dist/jquery.factory.js",
+			"dist/jquery.factory.slim.js"
+		],
+
+		languageOptions: {
+			ecmaVersion: 5,
```

Comment:
We should also verify minified files are ES5 - and only that: of course, we don't want to actually run any ESLint rules on minified code. I think we've been doing that before migrating to the flat config.